### PR TITLE
[Android]support TravisCI

### PR DESCRIPTION
**Pull request by katherine95s**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/apache/incubator-weex-playground.git"
+  },
+  "keywords": [
+    "weex",
+    "playground"
+  ],
+  "license": "ISC",
```
Comment (janpio) [inline package.json:16]: This should be `Apache-2.0`

Comment (katherine95s) [reply package.json]: The license has been corrected, thanks for reviewing!

Comment (katherine95s) [reply package.json]: > This should be `Apache-2.0`

The license has been corrected, thanks for reviewing!

Comment (katherine95s) [reply package.json:16]: The license has been corrected, thanks for reviewing!

**Pull request merged**